src/utils/fathom.ts

```diff
@@ -0,0 +1,6 @@
+import { fathom } from '../lib/fathom'
+
+export const recordClickInFathom = (eventId: string) => {
```

Comment:
**praise** absolutely love this!

**suggestion**: WDYT about naming this `trackFathomGoal` instead, to more closely align with their `fathom.trackGoal` api?

Reply:
agreed! I was renaming it a few times but couldn't think of the best name here, thanks!